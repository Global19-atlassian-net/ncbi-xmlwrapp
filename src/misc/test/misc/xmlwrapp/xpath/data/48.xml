<?xml version="1.0"?>
<root xmlns="http://bla.com">
    <child attr1="attr_val">value</child>
</root>
