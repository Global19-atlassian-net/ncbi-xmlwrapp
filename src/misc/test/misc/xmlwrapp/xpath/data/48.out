<?xml version="1.0"?>
<root xmlns="http://bla.com">
    <child attr1="attr_val">replaced_value</child>
</root>

